test/cmd/policy.sh

```diff
@@ -97,6 +97,25 @@ os::cmd::expect_success_and_not_text 'oc policy can-i --list --groups system:una
 os::cmd::expect_success_and_not_text 'oc policy can-i --list --user harold --groups system:authenticated' 'get update.*imagestreams/layers'
 os::cmd::expect_success_and_text 'oc policy can-i --list --user harold --groups system:authenticated' 'create get.*buildconfigs/webhooks'
 
+#psp review for system-admin
+os::cmd::expect_success_and_text 'oc policy subject-review -f ${OS_ROOT}/test/testdata/job.yaml -o=jsonpath={.status.AllowedBy.name}' 'anyuid'
+os::cmd::expect_success "oc login -u bob -p bob"
+os::cmd::expect_success_and_text 'oc whoami' 'bob'
+os::cmd::expect_success 'oc new-project bob'
+os::cmd::expect_success_and_text 'oc policy subject-review -f ${OS_ROOT}/test/testdata/job.yaml -o=jsonpath={.status.AllowedBy.name}' 'restricted'
+os::cmd::expect_success_and_text 'oc policy subject-review -f ${OS_ROOT}/test/testdata/two_jobs.yaml -o=jsonpath={.status.AllowedBy.name}' 'restrictedrestricted'
+os::cmd::expect_success_and_text 'oc policy review -f test/testdata/job.yaml -ojsonpath={.status.allowedServiceAccounts}' '\[\]'
+os::cmd::expect_failure 'oc policy subject-review -f ${OS_ROOT}/test/testdata/external-service.yaml'
+os::cmd::expect_success "oc login -u system:admin -n '${project}'"
+os::cmd::expect_success_and_text 'oc policy subject-review -u bob -g system:authenticated -f ${OS_ROOT}/test/testdata/job.yaml -n bob -o=jsonpath={.status.AllowedBy.name}' 'restricted'
+os::cmd::expect_success_and_text 'oc policy subject-review -u bob -f ${OS_ROOT}/test/testdata/job.yaml -n bob --no-headers=true' 'Not allowed'
+os::cmd::expect_success_and_text 'oc policy subject-review -z default -f ${OS_ROOT}/test/testdata/job.yaml' ''
+os::cmd::expect_success_and_text 'oc policy subject-review -z default -g system:authenticated -f ${OS_ROOT}/test/testdata/job.yaml' 'restricted'
+os::cmd::expect_success 'oc create -f ${OS_ROOT}/test/testdata/scc_lax.yaml'
+os::cmd::expect_success "oc login -u bob -p bob"
+os::cmd::expect_success_and_text 'oc policy review -f ./test/testdata/job.yaml --no-headers=true' 'default   lax'
+os::cmd::expect_success "oc login -u system:admin -n '${project}'"
+os::cmd::expect_success 'oc delete project bob'
```

Comment:
It would be nice if you could use different resources for each test, iow. job, deployment, just pod, etc.